### PR TITLE
[FIX] point_of_sale: Update certificate import

### PR DESCRIPTION
**Pull request by qle-odoo**
With the new version of chrome 95 we need adapt the documentation

### CODE REVIEW

File: content/applications/sales/point_of_sale/overview/epos_ssc.rst [hunk 1]
```diff
@@ -62,27 +62,24 @@ for example, `https://192.168.1.25`. Then, accept the self-signed certificate.
...
+Go to the *Details* tab and click on **Copy to file**.
+Select X.509 in base 64 and save it.
 
-.. image:: epos_ssc/certificate-details.png
-   :align: center
-   :alt: Details of the ePOS printer certificate
-
-Import the Self-signed certificate to your browser (Chrome)
+Import the Self-signed certificate to windows (Using Chrome)
 ===========================================================
```
Comment (Feyensv) [inline content/applications/sales/point_of_sale/overview/epos_ssc.rst:75]:
```suggestion
============================================================
```
Should be the same length as the title (--> `WARNING: Title underline too short` --> red runbot)

File: content/applications/sales/point_of_sale/overview/epos_ssc.rst [hunk 2]
```diff
@@ -62,27 +62,24 @@ for example, `https://192.168.1.25`. Then, accept the self-signed certificate.
...
-Go to the *Details* tab and click on **Export**.
+Go to the *Details* tab and click on **Copy to file**.
+Select X.509 in base 64 and save it.
 
-.. image:: epos_ssc/certificate-details.png
-   :align: center
-   :alt: Details of the ePOS printer certificate
-
-Import the Self-signed certificate to your browser (Chrome)
+Import the Self-signed certificate to windows (Using Chrome)
```
Comment (Feyensv) [inline content/applications/sales/point_of_sale/overview/epos_ssc.rst:74]:
```suggestion
Import the Self-signed certificate to Windows (Using Chrome)
```
?

File: content/applications/sales/point_of_sale/overview/epos_ssc.rst [hunk 3]
```diff
@@ -62,27 +62,24 @@ for example, `https://192.168.1.25`. Then, accept the self-signed certificate.
...
    :align: center
    :alt: The web browser indicates that the connection to the printer is not secure.
 
-Go to the *Details* tab and click on **Export**.
+Go to the *Details* tab and click on **Copy to file**.
+Select X.509 in base 64 and save it.
 
-.. image:: epos_ssc/certificate-details.png
-   :align: center
-   :alt: Details of the ePOS printer certificate
```
Comment (Feyensv) [inline content/applications/sales/point_of_sale/overview/epos_ssc.rst:75]: If the file is not used anymore (I suppose), it can be removed as well.

**Review by Feyensv — COMMENTED** [review]
@robodoo delegate+

File: content/applications/sales/point_of_sale/overview/epos_ssc.rst [hunk 4]
```diff
@@ -62,27 +62,24 @@ for example, `https://192.168.1.25`. Then, accept the self-signed certificate.
...
 In your Chrome browser, go to :menuselection:`Settings --> Privacy and security --> Security -->
-Manage certificates`, then open the *Authorities* tab, click on **Import**, and select all
-functionalities of the certificate.
+Manage certificates`
 
-.. tip::
-   You can directly access these settings by navigating to `chrome://settings/certificates`.
+Go to the *Trusted Root Certification Authorities* tab and click on **Import** and select
+your previous file.
+Accept all warning and restart your browser.
```
Comment (AntoineVDV) [inline content/applications/sales/point_of_sale/overview/epos_ssc.rst:82]:
```suggestion
Go to the *Trusted Root Certification Authorities* tab and click on **Import** and select
your previous file. Accept all warnings and restart your browser.
```
- This is misleading to return to a new line in rst while it's a single paragraph in html. If you intend to force a line break in html, prefix each line with a pipe `| ` in rst.
- warning**s**

**Review by AntoineVDV — COMMENTED** [review]
JCS is on holiday and doc code owners are not set up yet so we'll skip his review this time.

**Review by AntoineVDV — APPROVED** [review]

**Comment by qle-odoo** [comment]
@robodoo r+

**Pull request closed without merging**